config/configs/minimal.yaml

```diff
@@ -147,3 +147,7 @@ BLOB_SIDECAR_SUBNET_COUNT: 6
 # Whisk
 WHISK_EPOCHS_PER_SHUFFLING_PHASE: 4
 WHISK_PROPOSER_SELECTION_GAP: 1
+
+GENESIS_EPOCH: 0
+
+TIMELY_TARGET_FLAG_INDEX: 1
```

Comment:
These shouldn't be here. The difference between them and preset/config is that there can be multiple presets/configs. So the constants are true constants. We should add a `Constants` module with 0-arity functions like `genesis_epoch`, `timely_target_flag_index`, and so on, instead of reading them from a file.

Reply:
I agree, should this be done in this same pull request, or would it be better to open an independent issue?

Reply:
Doing it here should be fine since it's a small change

Reply:
I created  `lib/constants.ex`. Is that location OK?

Reply:
Yes 👍